### ISSUE TITLE
`notebook-view` appends URL to figure path somewhere and this error on Windows 

### ISSUE THREAD

**Issue by cderv** [quarto-dev/quarto-cli#4337]
From our `tests` folder 
````
quarto render docs\smoke-all\2023\01\05\notebook-preview-complex.qmd --to html
````
This throws an error 
````
ERROR: La syntaxe du nom de fichier, de répertoire ou de volume est incorrecte. (os error 123), stat 'docs\smoke-all\2023\01\05\https:\www.kaggle.com\code\kanncaa1\data-sciencetutorial-for-beginners'

Error: La syntaxe du nom de fichier, de répertoire ou de volume est incorrecte. (os error 123), stat 'docs\smoke-all\2023\01\05\https:\www.kaggle.com\code\kanncaa1\data-sciencetutorial-for-beginners'
    at Object.lstatSync (deno:runtime/js/30_fs.js:305:9)
    at existsSync (file:///C:/Users/chris/Documents/DEV_R/quarto-cli/src/vendor/deno.land/std@0.166.0/fs/exists.ts:75:10)
    at Array.filter (<anonymous>)
    at Object.complete (file:///C:/Users/chris/Documents/DEV_R/quarto-cli/src/command/render/render.ts:327:24)
    at async Object.onPostProcess (file:///C:/Users/chris/Documents/DEV_R/quarto-cli/src/command/render/render-files.ts:568:28)
    at async renderFiles (file:///C:/Users/chris/Documents/DEV_R/quarto-cli/src/command/render/render-files.ts:524:7)
    at async render (file:///C:/Users/chris/Documents/DEV_R/quarto-cli/src/command/render/render-shared.ts:101:18)
    at async Command.fn (file:///C:/Users/chris/Documents/DEV_R/quarto-cli/src/command/render/cmd.ts:193:26)
    at async Command.execute (file:///C:/Users/chris/Documents/DEV_R/quarto-cli/src/vendor/deno.land/x/cliffy@v0.25.4/command/command.ts:1790:7)
    at async quarto (file:///C:/Users/chris/Documents/DEV_R/quarto-cli/src/quarto.ts:111:3)
````

We are taking the url from `notebook-view` config 
https://github.com/quarto-dev/quarto-cli/blob/main/tests/docs/smoke-all/2023/01/05/notebook-preview-complex.qmd#L23-L26

and somehow create a path with it 
````
docs\smoke-all\2023\01\05\https:\www.kaggle.com\code\kanncaa1\data-sciencetutorial-for-beginners
````

which is a non valid PATH and fails on Windows when resources file are computed
https://github.com/quarto-dev/quarto-cli/blob/9e09dedabadc4a37bf4ff5aeb61862e3ca4a257d/src/command/render/render.ts#L326-L329

**Comment by cderv** [comment]
@dragonstyle seems related to `notebook-view` recent addition. This is for the embed feature right ?

**Comment by cderv** [comment]
Probably no need to join absolute `href` as if they were a local file I guess ? 
https://github.com/quarto-dev/quarto-cli/blob/9e09dedabadc4a37bf4ff5aeb61862e3ca4a257d/src/format/html/format-html-notebook.ts#L278-L282

**Comment by cderv** [comment]
@dragonstyle hope it helps find the quick fix, you know better than me this part so even if I may have found the right spot maybe. I'll let you fix this and merge the fix in #4099 - this should solve windows issue
